torch/_dynamo/variables/torch.py

```diff
@@ -721,6 +642,150 @@ def check_any_unspec(x):
 
             return tensor_variable
 
+    def _call_ntuple(self, tx, args, kwargs):
```

Comment:
Any changes from this code motion?

Reply:
No change, just move it to ```TorchInGraphFunctionVariable```.

Reply:
I hand audited, it looks not

Reply:
I still don't think I changed anything in this function after double checking.